lib/ticker.dart

```diff
@@ -68,54 +68,44 @@ mixin Tick {}
 
 @immutable
 class ImmutableTick extends BlocState with Tick {
-  const ImmutableTick();
```

Comment:
removed `hashCode` and `==` overrides because you want different instances of `ImmutableTick` to be treated as non-equal.